### ISSUE TITLE
Kill existing Julia session

### ISSUE THREAD

**Issue by ma-sadeghi** [JuliaPy/pyjulia#536]
Is there a way to kill the existing Julia session? I need this because I use `multiprocessing` to run a Julia function in parallel and for that to work, julia initialization must be done inside the function that's passed to `multiprocessing.Pool`. So far, everything works great, but I want to guard against the scenario in which a user (for some reason) has already initialized Julia in the calling script.

**Comment by mkitti** [comment]
Well we would want to interrupt Julia and then call the at exit hook before ending the process:

https://docs.julialang.org/en/v1/manual/embedding/

**Comment by ma-sadeghi** [comment]
Is this something that you might consider adding to pyjulia? Thanks!

**Comment by mkitti** [comment]
Frankly, https://pypi.org/project/juliacall/ is under much active development at the moment.

You can find the corresponding Github repository here:
https://github.com/JuliaPy/PythonCall.jl

**Comment by ma-sadeghi** [comment]
Apologies if I'm misunderstanding, but are you suggesting that I should possibly be using juliacall instead of pyjulia?

**Comment by mkitti** [comment]
Yes. You're asking for a new feature, and our capacity to work on new features for pyjulia are a bit limited at the moment.

Also have you considered doing the parallelization on the Julia side? You could use the Distributed model as described in https://docs.julialang.org/en/v1/manual/distributed-computing/

If you are doing parallelization all on a local computer, then you might want to consider multithreading rather than multiprocessing.
